CMakeLists.txt

```diff
@@ -295,6 +295,23 @@ if (ITKV3_COMPATIBILITY)
   message(FATAL_ERROR "ITKV3_COMPATIBILITY is removed starting in ITK version 5.0, this option is no longer valid.")
 endif()
 
+# Chose flavour of Analyze reader
+set(ITK_ANALYZE_FLAVOUR_DOC_STRING "Which flavour of Analyze reader to use by default")
+
+set(ITK_ANALYZE_FLAVOUR "ITK4Warning" CACHE STRING ${ITK_ANALYZE_FLAVOUR_DOC_STRING})
```

Comment:
ITK_ANALYZE_FLAVOUR name is a bit too generic (people who don't know that Analyze is an image file format may misinterpret it). Also, "flavour" is British spelling, while in ITK uses US speclling (it is always spelled as "flavor" in ITK). So, an option name like these could work better:
- ITK_ANALYZE_IMAGE_IO_FLAVOR
- ITK NIFTI_IO_ANALYZE_FLAVOR
- ITK NIFTI_IMAGE_IO_ANALYZE_FLAVOR